packages/mobile-top-nav/src/assets/img/icon_web.js

```diff
@@ -0,0 +1,43 @@
+import { html } from 'lit-element';
+
+export default (fillColor) => (
```

Comment:
Can you do these like you did with the Search and User icons below where you use the CSS class to change its styling?

Reply:
Yeah, those can be converted to components instead. I was at a point where I was considering generating an SVG use map instead, but I don't think those are accessible outside of the topnav element's shadowRoot so it wouldn't be useful to do so.